EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/SteamHandler/SteamMessageHandler.cs

```diff
@@ -5,35 +5,43 @@ namespace HK47.MessageHandlers;
 
 public class SteamMessageHandler(ISteamService steamService, IDataService dataService, IMessageService messageService) : ISteamMessageHandler
 {
-    /// </inheritdoc>
+    /// <inheritdoc/>
     public async Task ProcessCommand(string userInput)
     {
         var commandParams = userInput.Split(' ');
-        var command = commandParams!.First().ToLower();
+        var command = commandParams[0].ToLower();
         switch (command)
         {
-            case "requestgame":
-            case "gr":
-                try
-                {
-                    var appId = commandParams[1];
-                    var game = await steamService.RetrieveGameFromSteam(appId);
-                    await dataService.WriteGameToFile(game);
-                    await messageService.SendMessage($"```[Statement]: {game.Name} was added to the list.```");
-                }
-                catch (Exception ex)
-                {
-                    await messageService.SendMessage($"[Error]: Game not added due to the following:\n{ex.Message}");
-                }
+            case "requestGame":
+                await RecordGameRequest(commandParams[1]);
```

Comment:
also just checking we wanna pass the [1] and not the params generally like we are doing for await RememberWhen(commandParams);

Reply:
I think that is completely dependent on the command that is being run. The reason RememberWhen(...) requires the command params is because it takes the commands "remember" AND "remember when".
This inherently means that the method has two distinct functionalities that are dependent on the different arguments present.